### ISSUE TITLE
Missing distributionformat in output HNAP XML

### ISSUE THREAD

**Issue by alexandreleroux** [geopython/pygeometa#23]
Potential bug, or at least a parameter missing from the output HNAP XML. Under gmd:MD_Distribution, there should be a gmd:distributionFormat. Currently, distributionformat is nowhere in pygeometa's output XML.

Example I've been provided:
```
  <gmd:distributionInfo>
    <gmd:MD_Distribution>
      <gmd:distributionFormat>
        <gmd:MD_Format>
          <gmd:name>
            <gco:CharacterString>dsa</gco:CharacterString>
          </gmd:name>
          <gmd:version>
            <gco:CharacterString>sfda</gco:CharacterString>
          </gmd:version>
        </gmd:MD_Format>
      </gmd:distributionFormat>
...
```
Let me know if you need more details, thanks! -- Alex

**Comment by tomkralidis** [comment]
Fixed in master in #29.  Note that for `distribution:*` sections, if `format` and `format_version` parameters exist, they will be emitted on output.  @alexandreleroux can you add to docs?

**Comment by alexandreleroux** [comment]
Updated doc in commit 363d80b994aaa4f6603840f536220bd1830b5d82 , closing ticket
